utils/generate-css.ts

```diff
@@ -0,0 +1,15 @@
+export function generatecss(stylesList: string[]) {
+  let ans = ''
+  let ansArr: string[] = []
+  stylesList.forEach((style) => {
+    ans =
+      `'` +
+      style.split(' ').slice(0, -4).join(' ') +
+      `'` +
+      ', ' +
+      style.split(' ')[style.split(' ').length - 1]
+    if (!ansArr.includes(ans)) ansArr.push(ans)
```

Comment:
Consider using `{}` for closing `if` or `for` statement. It would help to be easier to follow the code.

```suggestion
    if (!ansArr.includes(ans)) {
        ansArr.push(ans)
    }
```